kyo-prelude/shared/src/main/scala/kyo/Batch.scala

```diff
@@ -83,7 +83,7 @@ object Batch:
       *   A batched operation that produces a single value from the sequence
       */
     inline def eval[A](seq: Seq[A])(using inline frame: Frame): A < Batch[Any] =
-        ArrowEffect.suspend[A](erasedTag[Any], Op.Eval(seq))
+        ArrowEffect.suspend[A](erasedTag[Any], seq)
```

Comment:
Suspend with the `Seq` directly to avoid the `Eval` wrapper allocation. See change to make `Op` a type union at the end of the file.